Project.toml

```
@@ -1,6 +1,6 @@
 name = "DataStructures"
 uuid = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
-version = "0.19.0-DEV"
+version = "0.19.1-DEV"
```

```suggestion
version = "0.19.0-DEV"
```
No need to increment bwhen version bus suffixes with DEV and the digit you would increment is currently 0